go/interpreter/lfvm/instructions.go

```diff
@@ -1118,8 +1098,6 @@ func opCall(c *context) {
 
 	// first use static and dynamic gas cost and then resize the memory
 	// when out of gas is happening, then mem should not be resized
-	c.memory.EnsureCapacityWithoutGas(needed_memory_size, c)
```

Comment:
Removing this line here has the same problem as when it was removed in the create call.